src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReferralHelperTest.java

```diff
@@ -111,8 +111,8 @@ class ReferralHelperTest {
 
     private final String expectedCreatedReferralReply = "ReferralReplyType(directionTo=directionTo, replyToEmailAddre"
         + "ss=replyTo, isUrgentReply=isUrgent, directionDetails=directionDetails, replyDocument=null, replyGeneralNot"
-        + "es=generalNotes, replyBy=Judge Alex, replyDate=" + Helper.getCurrentDate()
-        + ", replyDateTime=null, referralSubject=null)";
+        + "es=generalNotes, replyBy=Judge Alex, replyDate=" + Helper.getCurrentDate() + ", replyDateTime="
+        + Helper.getCurrentDateTime().substring(0, 11) + ", referralSubject=Other)";
```

Comment:
why are we substringing this?

Reply:
It was to make comparing dates easier. It has been updated to do a comparison of real dates.